openapi-spec/swagger.yaml

```diff
@@ -1530,15 +1530,12 @@ definitions:
     description: >-
       Extends the size of a volume to a requested size, in gibibytes (GiB).
     type: object
-    properties:
-      extend:
-        type: object
-        required:
+    required:
```

Comment:
In the code I didn't see `ExtendVolumeRequest` schema:
```go
// ExtendVolumeSpec ...
type ExtendVolumeSpec struct {
	NewSize int64 `json:"newSize,omitempty"`
}
```

Please keep the format same with what is in the code.